build.gradle.kts

```diff
@@ -0,0 +1,123 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
What was the motivation for replacing the build script with Kotlin? The included `build.gradle` I believe was autogenerated when setting up the project, has more Ghidra stuff been moved to Kotlin?